README.md

```diff
@@ -1,31 +1,31 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
 # Project Name
+Portfolio
 
-> One paragraph statement about the project.
+> This is my portfolio project.
 
 ![screenshot](./app_screenshot.png)
 
 Additional description about the project and its features.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
+- CSS
+- Linters
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://gemmen29.github.io/Portfolio/)
```

Comment:
- [ ] [OPTIONAL] Nice job adding the live Demo 👍 Right now however it is just showing the Readme file in a separate tab. if you are interested I suggest you have a look at [this article](https://lirad.medium.com/two-ways-to-create-a-live-demo-for-your-github-project-5436b048a40e) which explains how you can setup an actual live Demo of your webpage